helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java

```diff
@@ -523,6 +532,27 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
     }
   }
 
+  private static Map<String, String> parseListOfPairs(List<String> list, String fieldName) {
```

Comment:
There's a `argToStringMap` method in `com.spotify.helios.cli.Utils` that looks similar, AFAICT. Would be nice to kill off one of them maybe. (I'm not very fond of `argToStringMap`).